utils/OffchainStepper.js

```diff
@@ -287,13 +305,30 @@ module.exports = class OffchainStepper extends VM.MetaVM {
       isCallDataRequired = true;
     }
 
+    const codeReads = codeProof.readAndWrites.filter(
+      function (val) {
+        return val < runState.code.length;
+      }
+    ).sort(
+      function (a, b) {
```

Comment:
isn't this a normal sort?

Reply:
The normal sort converts the values to `String` and this means:
```
> [2,3,4,5,9,1].sort()
[ 1, 2, 3, 4, 5, 9 ]
> [2,71,4,5,9,1,].sort()
[ 1, 2, 4, 5, 71, 9 ]
```
😅 😄

Reply:
Thank you. I see :rofl: